tinyfl/aggregator/__init__.py

```diff
@@ -55,6 +55,17 @@
 with open(sys.argv[1]) as f:
     config = json.load(f)
     host, port = itemgetter("host", "port")(config)
+    consensus, timeout, epochs = itemgetter("consensus", "timeout", "epochs")(config)
```

Comment:
Add checks for all the variables being loaded from the config file. Also confirm that the data types are correct? I had to  cast `port` to `int` in line 201.